package.json

```diff
@@ -5,10 +5,24 @@
   "license": "MIT",
   "private": true,
   "workspaces": [
-    "packages/**/*"
+    "packages/**/*",
+    "docs/**/*"
   ],
   "packageManager": "yarn@4.5.1",
   "devDependencies": {
-    "node-gyp": "^10.2.0"
+    "html-webpack-plugin": "^5.6.3",
+    "node-gyp": "^10.2.0",
+    "typescript": "^5.6.3",
+    "webpack": "^5.96.1",
+    "webpack-cli": "^5.1.4",
+    "webpack-dev-server": "^5.1.0"
+  },
+  "scripts": {
+    "build": "yarn run build:providers && yarn run build:widgets",
+    "build:providers": "yarn workspace @webex/widget-provider build:src",
+    "build:widgets": "yarn workspace @webex/widget-cc-station-login build:src && yarn workspace @webex/widget-cc-user-state build:src",
+    "samples:build": "yarn workspace @webex/react-samples-app build:src && yarn workspace @webex/web-component-samples-app build:src",
+    "samples:serve-react": "yarn workspace @webex/react-samples-app serve",
+    "samples:serve-wc": "yarn workspace @webex/web-component-samples-app serve"
```

Comment:
Non-blocking: Can we have one serve command that shows both wc as well as react apps?

Reply:
https://github.com/user-attachments/assets/1c988e62-719e-4274-b039-c6a21f5dcfd4

This is kind of hacky

What I'm doing?
- Running the react-samples-app in the background (this is at localhost:300)
- Opening the home page 

What happens when we click on the buttons?
- When we click on "Contact Center in a react app", I'm re-directly to localhost:3000
- When we click on "Contact Center widgets as web components" - Im opening the html file that contains the web-component

Problem 
- Not sure if we will be able to run this in github pages
- Not sure if this is a correct way of running both the samples app from the same page

Reply:
Now that I think about it maybe we should have another application that imports both react and web-component app and then just renders them. And we can run this app using webpack-dev-server